components/header.js

```diff
@@ -89,12 +92,21 @@ const Header = () => {
             top: 0,
             left: 0,
             right: 0,
+            height: '100%',
             zIndex: -1
           }}
         >
           <source
-            src="https://hackclub.github.io/summer-slack-video/slack.mp4"
-            type="video/mp4"
+            src="https://cdn.glitch.com/3899929b-9aed-4dae-b1e6-230ef0ed4d51%2Fslack.hevc.mp4?v=1590694006597"
```

Comment:
How does this work? Are you just uploading them straight to glitch?

Reply:
Yeah, just in the assets section. Not ideal for perf since a separate domain but storing big files in Git is such a mess & so bloated…idk what the best solution is